net/wallet/Portfile

```diff
@@ -0,0 +1,239 @@
+                    patch-rename-server-wallet-admin.diff \
+                    patch-rename-server-wallet-backend.diff \
+                    patch-rename-server-wallet-report.diff \
+                    patch-tests-client-full-t.in.diff \
+                    patch-tests-client-prompt-t.in.diff
+use_autoreconf      yes
+
+# Use configure, and pass in the paths to MacPorts kerberos5, remctl, and perl.
+configure.args      --enable-reduced-depends \
+                    --with-remctl=${prefix} \
+                    PATH_KRB5_CONFIG=${prefix}/bin/krb5-config \
+                    PERL=${prefix}/bin/perl${perl5.major} 
+
+# The Wallet client is just a couple of binaries, plus man pages and docs.
+variant client description 'Wallet client' {
+    long_description-append The wallet client uses the remctl protocol to \
+                            communicate with a wallet server.
+}
+
+# The Wallet server is entirely Perl, spawned by the remctl daemon.
+# TODO: Patch wallet remctl config files to use ${destroot}${prefix}
+variant server description 'Wallet server' {
+    # We can't use perl5.setup because it overrides alot of settings that
+    # we need.
+    #perl5.setup		Wallet 1.003
+
+    long_description-append The wallet server, run by remctld, maintains \
+                            the database of object metadata and secure \
+                            objects, and responds to requests from the \
+                            wallet client.
+
+    if {${perl5.major} != ""} {
+        depends_lib-append      port:p${perl5.major}-datetime \
+                                port:p${perl5.major}-dbi \
+                                port:p${perl5.major}-dbix-class \
+                                port:p${perl5.major}-digest-md5 \
+                                port:p${perl5.major}-sql-translator \
+                                port:p${perl5.major}-timedate
+        depends_build-append    port:p${perl5.major}-crypt-generatepassword \
+                                port:p${perl5.major}-datetime-format-sqlite
+    }
+
+    # TODO: Tests have a Stanford-specific part, and a NetDB-verifier part,
+    # which need to be disabled.
+    # test.run      yes
+    # test.target   check
+
+notes-append "
+**
+* Before using the Wallet server, you will need to choose a database
+* backend to use.  MySQL, Postgres, and SQLite are known to work.
+* Then you will need to install the p5-datetime-format-* and p5-dbd-*
+* ports that match the database backend you chose.
+*
+* If you want to support getting keytabs through Wallet, then your KDC
+* will need to have the wallet port installed with the +kdc variant.
+*
+* Other Perl modules may be required, depending on what you want to
+* support.  Read ${prefix}/share/doc/wallet/setup
+* for additional server configuration instructions.
+*
+* Wallet server runs via remctl, so be sure that remctld is running,
+* and is configured correctly!
+**
```

Comment:
We try to not have special marker symbols in front of the messages. If they need to be visually separated from the rest of the output, that's something MacPorts itself should do (and the current master already does that).